modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/identity/mgt/AccountLockEnabledTestCase.java

```diff
@@ -30,60 +29,62 @@
+    private static final String CONNECTOR_ACCOUNT_LOCK_HANDLER = "YWNjb3VudC5sb2NrLmhhbmRsZXI";
+    private static final String LOCALE_ATTRIBUTE = "locale";
+    private static final String USERS_PATH = "users";
+    private static final String USER_SCHEMA = "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User";
 
-    private String defaultLocalityClaimUri = IdentityUtil.getClaimUriLocale();
-    private String accountLockClaimUri = "http://wso2.org/claims/identity/accountLocked";
-    private String defaultLocalityClaimValue = "en_us";
-    private String registryResourcePath = "/_system/config/identity/email/";
-
-    private String testLockUser1 = "TestLockUser1";
-    private String testLockUser1Password = "TestLockUser1Password";
-    private String testLockUser1WrongPassword = "TestLockUser1WrongPassword";
-    private String testLockUser2 = "TestLockUser2";
-    private String testLockUser2Password = "TestLockUser2Password";
-    private String testLockUser3 = "TestLockUser3";
-    private String testLockUser3Password = "TestLockUser3Password";
-
-    private String accountLockTemplateWhenUserExceedsFailedAttempts = "accountlockfailedattempt";
-    private String accountLockTemplateWhenAdminTriggered = "accountlockadmin";
-    private String accountUnlockTemplateAdminTriggered = "accountunlockadmin";
-    private String accountUnlockTemplateTimeBased = "accountunlocktimebased";
-
-    private AuthenticatorClient authenticatorClient;
-    private RemoteUserStoreManagerServiceClient usmClient;
+
+    private SCIM2RestClient scim2RestClient;
+    private AuthenticatorRestClient authenticatorRestClient;
     private EmailTemplatesRestClient emailTemplatesRestClient;
     private IdentityGovernanceRestClient identityGovernanceRestClient;
     private ConnectorsPatchReq connectorPatchRequest;
 
-    private static final String ENABLE_ACCOUNT_LOCK = "account.lock.handler.lock.on.max.failed.attempts.enable";
-    private static final String CATEGORY_LOGIN_ATTEMPTS_SECURITY = "TG9naW4gQXR0ZW1wdHMgU2VjdXJpdHk";
-    private static final String CONNECTOR_ACCOUNT_LOCK_HANDLER = "YWNjb3VudC5sb2NrLmhhbmRsZXI";
 
-    private static final String TRUE_STRING = "true";
-    private static final String DEFAULT = "default";
-    private static final String USER_LOCALE = "en_US";
+    private String testLockUserId;
+    private String testLockUser2Id;
+    private String testLockUser3Id;
 
     @SetEnvironment(executionEnvironments = {ExecutionEnvironment.ALL})
     @BeforeClass(alwaysRun = true)
     public void testInit() throws Exception {
         super.init();
-        authenticatorClient = new AuthenticatorClient(backendURL);
-        enableAccountLocking(ENABLE_ACCOUNT_LOCK);
-        usmClient = new RemoteUserStoreManagerServiceClient(backendURL, sessionCookie);
+        authenticatorRestClient = new AuthenticatorRestClient(backendURL.replace("services/", ""));
```

Comment:
Since majority of the rest cases will need the backend URL (without the /services part), we can look into better ways to get it.

Reply:
Fixed with https://github.com/wso2/product-is/pull/15906/commits/58e6cec24ce802db3f770b7bbac811a582bb02c2